### PR TITLE
fix footer position

### LINKED ISSUE #19: Make the footer on home page responsive
The footer should stick to bottom of screen for all screen sizes

### PR DESCRIPTION
**Pull request by imanpalsingh**
Fixes https://github.com/imabhinavagarwal/ComponentLibrary/issues/19

### CODE REVIEW

**Comment by imanpalsingh** [comment]
@imabhinavagarwal The entire site is not responsive, if you want to make just the footer more responsive (like for mobile devices) you'll have to fix lot of other things as well. Anyways, this PR always keeps it at bottom as mentioned.

**Comment by imabhinavagarwal** [comment]
@imanpalsingh Can you elaborate more about rest part being not responsive? 
Since this library is made only with html and css and no javascript, I guess there may be a few unresponsive parts.
But if you feel like those could also be fixed, I can create another issue and maybe you can help to fix rest things as well?

Actually, I started this as a learning project long back and now after working with FE I am looking to expand the scope and try to make it as useful as possible instead of it just being a dummy repo.

File: main.css [hunk 1]
```diff
@@ -100,7 +100,7 @@ button {
     color: var(--white);
     width: 100%;
     bottom: 0;
-    position: relative;
+    position: fixed;
```
Comment (imabhinavagarwal) [inline main.css:103]: Weird.
I tried the same previously but didn't work for me and now it seems to be working fine. I guess when I tried this, other css classes may have different values. Anyway, thanks for fixing this. this was bugging me for a long time

**Review by imabhinavagarwal — APPROVED** [review]

**Comment by imanpalsingh** [comment]
I tried the page on a smaller screen device, the UI was not responsive. It might need a bit of css rewrite. Anyways feel free to create new issues if you need help, I'll be happy to help.

**Pull request merged**